src/main/java/io/vertx/ext/hawkular/impl/Sender.java

```diff
@@ -72,6 +75,24 @@ public Sender(Vertx vertx, VertxHawkularOptions options, Context context) {
     context.runOnContext(aVoid -> {
       httpClient = vertx.createHttpClient(options.getHttpOptions());
       timerId = vertx.setPeriodic(MILLISECONDS.convert(batchDelay, NANOSECONDS), this::flushIfIdle);
+
+      // Configure the metrics bridge. It just transforms the received metrics (json) to a Single Metric to enqueue it.
+      if (options.isMetricsBridgeEnabled() && options.getMetricsBridgeAddress() != null) {
+        vertx.eventBus().consumer(options.getMetricsBridgeAddress(), message -> {
+          // By spec, it is a json object.
+          JsonObject json = (JsonObject) message.body();
+
+          // source and value has to be set.
+          // the timestamp can have been set in the message using the 'timestamp' field. If not use 'now'
+          // the type of metrics can have been set in the message using the 'type' field. It not use 'gauge'
+          SingleMetric metric = new SingleMetric(json.getString("source"),
+              json.getLong("timestamp", System.currentTimeMillis()),
+              json.getDouble("value"),
+              MetricType.from(json.getString("type", MetricType.GAUGE.name())));
```

Comment:
I would avoid to use the string format imposed by `MetricType` in user code. This MetricType is going to be deprecated, because it was created to decode `statsd` messages in `ptrans` but then was used for all sort of purposes.

I think using `gauge` or `counter` type names in user code is a better option.